### ISSUE TITLE
parallelize dspot on Kubernetes

### ISSUE THREAD

**Issue by monperrus** [STAMP-project/dspot#802]
it would be great to parallelize Dspot. Kubernetes is a nice technology for this.

Parallelization could be done at the test level or amplifier level.

(under discussion at KTH)

**Comment by Tailp** [comment]
Hi @monperrus and @danglotb  :), I was previously a bit hesitant about if I should make a scanner or modifying our repairnator project scanner so that it can also submit builds ids to dspot . In the end I decided to implement the our current k8s repairnator design fully also for Dspot in order to keep these two projects independent from each other without suffering any drawbacks. Design-wise I have made that k8s Dspot is identical to k8s repairnator so we can easily dismantle it and only reuse some of its components in the context of deploying this together with dspot for a more compact joined design. :) . More details can be found at the README of this [fork](https://github.com/Tailp/dspot/tree/kubernetes/kubernetes-support)

That's about the summary of the progess of this task. Some other things need to be done is maybe to break down [this PR](https://github.com/STAMP-project/dspot/pull/832) since it has becomes quite fat. Otherwise , ping me if any problems arise. Thank you :) .
